tests/core/encoding/ini/test_core_ini.odin

```diff
@@ -0,0 +1,120 @@
+package test_core_ini
+
+import "base:runtime"
+import "core:encoding/ini"
+import "core:mem/virtual"
+import "core:strings"
+import "core:testing"
+
+@test
+prase_ini :: proc(t: ^testing.T) {
+	ini_data := `
+		[LOG]
+		level = "devel"
+		file = "/var/log/testing.log"
+
+		[USER]
+		first_name = "John"
+		surname = "Smith"
+	`
+
+	m, err := ini.load_map_from_string(ini_data, context.allocator)
+	ini.delete_map(m)
```

Comment:
Did you mean `defer ini.delete_map(m)`? It seems to me that the tests below would effectively otherwise be a use-after-free error.

Reply:
Yes, I most definitely did, I have corrected this in the most recent commit.